diff -up efl-1.21.1/src/lib/evas/Evas_GL.h.khrplatform efl-1.21.1/src/lib/evas/Evas_GL.h
--- efl-1.21.1/src/lib/evas/Evas_GL.h.khrplatform	2019-02-15 19:58:49.927937159 -0500
+++ efl-1.21.1/src/lib/evas/Evas_GL.h	2019-02-15 19:58:55.970818084 -0500
@@ -4273,8 +4273,9 @@ typedef signed int       GLfixed;      /
 #ifndef GL_ES_VERSION_2_0
 /* GL types for handling large vertex buffer objects */
 #include <stddef.h>
-typedef ptrdiff_t GLintptr;     // Changed khronos_intptr_t
-typedef ptrdiff_t GLsizeiptr;   // Changed khronos_ssize_t
+#include <KHR/khrplatform.h>
+typedef khronos_intptr_t GLintptr;
+typedef khronos_ssize_t GLsizeiptr;
 #endif
 
 /* Some definitions from GLES 3.0.
