diff -up efl-1.22.2/src/lib/efl/interfaces/efl_ui_format.c.pragma efl-1.22.2/src/lib/efl/interfaces/efl_ui_format.c
--- efl-1.22.2/src/lib/efl/interfaces/efl_ui_format.c.pragma	2019-05-02 15:05:56.681118361 -0400
+++ efl-1.22.2/src/lib/efl/interfaces/efl_ui_format.c	2019-05-02 15:07:48.641683821 -0400
@@ -124,7 +124,10 @@ _default_format_cb(void *data, Eina_Strb
      }
    else if (sd->format_type == FORMAT_TYPE_STATIC)
      {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-security"
         eina_strbuf_append(str, sd->template);
+#pragma GCC diagnostic pop
      }
    else
      {
