diff -up efl-1.18.0/configure.ac.tslibfix efl-1.18.0/configure.ac
--- efl-1.18.0/configure.ac.tslibfix	2016-08-29 15:35:41.784900935 -0400
+++ efl-1.18.0/configure.ac	2016-08-29 15:35:48.936847061 -0400
@@ -3442,7 +3442,7 @@ EFL_INTERNAL_DEPEND_PKG([ECORE_FB], [eo]
 EFL_INTERNAL_DEPEND_PKG([ECORE_FB], [efl])
 EFL_INTERNAL_DEPEND_PKG([ECORE_FB], [eina])
 
-EFL_OPTIONAL_DEPEND_PKG([ECORE_FB], [${want_tslib}], [TSLIB], [tslib])
+EFL_OPTIONAL_DEPEND_PKG([ECORE_FB], [${want_tslib}], [TSLIB], [tslib-0.0])
 EFL_ADD_FEATURE([ECORE_FB], [tslib])
 
 EFL_EVAL_PKGS([ECORE_FB])
