diff -up efl-1.13.1/src/bin/edje/epp/cppalloc.c.noaddy efl-1.13.1/src/bin/edje/epp/cppalloc.c
--- efl-1.13.1/src/bin/edje/epp/cppalloc.c.noaddy	2015-02-27 15:17:31.361456345 -0500
+++ efl-1.13.1/src/bin/edje/epp/cppalloc.c	2015-02-27 15:17:40.397389751 -0500
@@ -16,8 +16,7 @@
  * GNU General Public License for more details.
  * 
  * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
  * 
  * In other words, you are welcome to use, share and improve this program.
  * You are forbidden to forbid anyone else to use, share and improve
diff -up efl-1.13.1/src/bin/edje/epp/cpperror.c.noaddy efl-1.13.1/src/bin/edje/epp/cpperror.c
--- efl-1.13.1/src/bin/edje/epp/cpperror.c.noaddy	2015-02-27 15:15:41.334267241 -0500
+++ efl-1.13.1/src/bin/edje/epp/cpperror.c	2015-02-27 15:16:50.293759012 -0500
@@ -16,8 +16,7 @@
  * GNU General Public License for more details.
  * 
  * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
  * 
  * In other words, you are welcome to use, share and improve this program.
  * You are forbidden to forbid anyone else to use, share and improve
diff -up efl-1.13.1/src/bin/edje/epp/cppexp.c.noaddy efl-1.13.1/src/bin/edje/epp/cppexp.c
--- efl-1.13.1/src/bin/edje/epp/cppexp.c.noaddy	2015-02-27 15:17:02.579668466 -0500
+++ efl-1.13.1/src/bin/edje/epp/cppexp.c	2015-02-27 15:17:20.407537075 -0500
@@ -13,8 +13,7 @@
  * GNU General Public License for more details.
  * 
  * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
  * 
  * In other words, you are welcome to use, share and improve this program.
  * You are forbidden to forbid anyone else to use, share and improve
diff -up efl-1.13.1/src/bin/edje/epp/cpphash.c.noaddy efl-1.13.1/src/bin/edje/epp/cpphash.c
--- efl-1.13.1/src/bin/edje/epp/cpphash.c.noaddy	2015-02-27 15:18:45.989906337 -0500
+++ efl-1.13.1/src/bin/edje/epp/cpphash.c	2015-02-27 15:18:55.706834724 -0500
@@ -16,8 +16,7 @@
  * GNU General Public License for more details.
  * 
  * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
  * 
  * In other words, you are welcome to use, share and improve this program.
  * You are forbidden to forbid anyone else to use, share and improve
diff -up efl-1.13.1/src/bin/edje/epp/cpplib.c.noaddy efl-1.13.1/src/bin/edje/epp/cpplib.c
--- efl-1.13.1/src/bin/edje/epp/cpplib.c.noaddy	2015-02-27 15:18:24.378065615 -0500
+++ efl-1.13.1/src/bin/edje/epp/cpplib.c	2015-02-27 15:18:35.423984207 -0500
@@ -16,8 +16,7 @@
  * GNU General Public License for more details.
  * 
  * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
  * 
  * In other words, you are welcome to use, share and improve this program.
  * You are forbidden to forbid anyone else to use, share and improve
diff -up efl-1.13.1/src/bin/edje/epp/cpplib.h.noaddy efl-1.13.1/src/bin/edje/epp/cpplib.h
--- efl-1.13.1/src/bin/edje/epp/cpplib.h.noaddy	2015-02-27 15:17:49.679321344 -0500
+++ efl-1.13.1/src/bin/edje/epp/cpplib.h	2015-02-27 15:18:01.774232204 -0500
@@ -13,8 +13,7 @@ MERCHANTABILITY or FITNESS FOR A PARTICU
 GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
-along with this program; if not, write to the Free Software
-Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+along with this program; if not, see <http://www.gnu.org/licenses/>.
 
  In other words, you are welcome to use, share and improve this program.
  You are forbidden to forbid anyone else to use, share and improve
diff -up efl-1.13.1/src/bin/edje/epp/cppmain.c.noaddy efl-1.13.1/src/bin/edje/epp/cppmain.c
--- efl-1.13.1/src/bin/edje/epp/cppmain.c.noaddy	2015-02-27 15:15:04.640537672 -0500
+++ efl-1.13.1/src/bin/edje/epp/cppmain.c	2015-02-27 15:15:27.094372189 -0500
@@ -14,8 +14,7 @@
  * GNU General Public License for more details.
  * 
  * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
  * 
  * In other words, you are welcome to use, share and improve this program.
  * You are forbidden to forbid anyone else to use, share and improve
