diff -up efl-1.19.0/src/lib/edje/edje_private.h.luajitfix efl-1.19.0/src/lib/edje/edje_private.h
--- efl-1.19.0/src/lib/edje/edje_private.h.luajitfix	2017-05-15 09:53:45.314265022 -0400
+++ efl-1.19.0/src/lib/edje/edje_private.h	2017-05-15 09:55:13.508048012 -0400
@@ -33,6 +33,9 @@
 #include <lua.h>
 #include <lualib.h>
 #include <lauxlib.h>
+#ifndef luaL_reg
+# define luaL_reg	luaL_Reg
+#endif
 #include <setjmp.h>
 
 #ifdef HAVE_EVIL
diff -up efl-1.19.0/src/lib/elua/elua_private.h.luajitfix efl-1.19.0/src/lib/elua/elua_private.h
--- efl-1.19.0/src/lib/elua/elua_private.h.luajitfix	2017-05-15 09:56:03.481791776 -0400
+++ efl-1.19.0/src/lib/elua/elua_private.h	2017-05-15 09:56:14.867505561 -0400
@@ -27,6 +27,9 @@
 #include <lua.h>
 #include <lualib.h>
 #include <lauxlib.h>
+#ifndef luaL_reg
+# define luaL_reg	luaL_Reg
+#endif
 
 #include "Elua.h"
 
diff -up efl-1.19.0/src/lib/evas/filters/evas_filter_parser.c.luajitfix efl-1.19.0/src/lib/evas/filters/evas_filter_parser.c
--- efl-1.19.0/src/lib/evas/filters/evas_filter_parser.c.luajitfix	2017-05-15 09:56:44.449766382 -0400
+++ efl-1.19.0/src/lib/evas/filters/evas_filter_parser.c	2017-05-15 09:57:00.786361265 -0400
@@ -13,6 +13,9 @@
 #include <lua.h>
 #include <lualib.h>
 #include <lauxlib.h>
+#ifndef luaL_reg
+# define luaL_reg	luaL_Reg
+#endif
 
 #define FILTERS_LEGACY_COMPAT
 
